### ISSUE TITLE
Microsoft Graph permission names

### ISSUE THREAD

**Issue by Sealjay-clj** [integrations/microsoft-teams#135]
**Describe the bug**
The [GItHub permission scopes](https://github.com/integrations/microsoft-teams#github-permission-scopes) listed do not list out the specific [Microsoft Graph permissions](https://docs.microsoft.com/en-us/graph/permissions-reference#microsoft-graph-permission-names) requested by the application.

This makes it hard to complete a security review.

**Expected behavior**
Expected that the documentation would include detail suitable to light up the app.

**Comment by gauravsaralMs** [comment]
Assigning to @ashokirla as looks like documentation issue.

**Comment by Madhava-gollapalli** [comment]
Yes, it is hard to assess the security review for this integration.

Is there any timeline for this to update into documentation.
